WordPress/src/main/java/org/wordpress/android/ui/accounts/UnifiedLoginTracker.kt

```diff
@@ -0,0 +1,64 @@
+package org.wordpress.android.ui.accounts
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.analytics.AnalyticsTracker.Stat.UNIFIED_LOGIN_STEP
+import org.wordpress.android.ui.accounts.UnifiedLoginTracker.Source.DEFAULT
+import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class UnifiedLoginTracker
+@Inject constructor(private val analyticsTracker: AnalyticsTrackerWrapper) {
+    private var currentSource: Source = DEFAULT
+    private var currentFlow: Flow? = null
+    fun track(step: Step) {
+        track(step = step)
+    }
+
+    fun track(flow: Flow? = null, step: Step) {
+        currentFlow = flow
+        if (BuildConfig.UNIFIED_LOGIN_AVAILABLE) {
+            currentFlow?.let {
```

Comment:
What do you think of logging an error here when `currentFlow` is null so we are able to know we're missing some flows? Or maybe `currentFlow` could have a default value, like "unknown"? I'll leave this up to you.

Reply:
I've originally had a crash here to find out when it's not working. I can still add a crash in debug and log an error. I'm afraid just logging an error will not give us much information. What do you think?

Reply:
I like this approach, thank you!

Reply:
I really like this approach, thank you!